.travis.yml

```diff
@@ -41,4 +41,4 @@ matrix:
         - docker-compose up -d --build
         - npm install frontend/
       script:
-        - docker-compose run --rm frontend npm test src/tests/ -- --coverage
+        - docker-compose run -e CI=true --rm frontend npm test src/tests/
```

Comment:
Did you remove the code coverage analysis on purpose?

Reply:
restored in 82cb072